ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfig.java

```diff
@@ -191,6 +191,9 @@ default int getMillisPerSlot() {
 
   int getReorgParentWeightThreshold();
 
+  // Handle spec tests with BLS disabled
+  boolean isBlsDisabled();
```

Comment:
NIT: Can the double-negative be misleading? Usually we default to `isSomethingEnabled`. Not sure what is the answer in this case, just thought I'd raise the question :)

Reply:
I've got a fairly strong dislike of this kind of code where theres 'test only' stuff in production code. i think i'd make a test class that disables bls in preference to changing a production class to not use BLS.

Reply:
should be largely addressed by #9805 if looking for the pr i merged